### DISCUSSION TITLE
Gerber and Ascii drill files

### DISCUSSION THREAD

**Discussion by Juanko84** [taligentx/dscKeybusInterface#282] [dscKeybusInterface Discussion]
Hi, 

I sent the included pcb files in the download to a manufacturer. They came back and said they cant use the files, they need the Gerber and Ascii drill files. Any chance this is available and can be sent to me please?

**Comment by taligentx** [comment]
Hi @Juanko84 - the PCB files were contributed by @sj-louw in #21, #22, #23, and #24.  I haven't looked into the manufacturing aspect - is there a way to convert the included files to the format needed?

**Comment by Juanko84** [comment]
Hi Nikhil,

I did convert the files to be used by pcb manufacturers. I will forward them.

I have a Question on the DSC Status, I see it can be used as a topic, but its not implemented, how can I use this status as a home assistant property?


I have an envisalink aswell connected to my dsc alarm and I get the below dsc status, can this be achieved with the availability_topic: "dsc/Status" in your setup? Have you got an example for this?


[cid:image001.jpg@01D68500.5A2CFBC0]

[cid:image005.jpg@01D68500.5A2CFBC0]

**Comment by taligentx** [comment]
Hi @Juanko84 - good to hear about the board files!  If they work out and you'd like to contribute, you can add a pull request with the converted files or attach the files here.

Your screenshots aren't showing up, what kind of information are you looking for in the dsc/Status MQTT topic?  Currently the examples send "online" and "offline" when the Keybus is connected/disconnected.

**Comment by Juanko84** [comment]
Hi @taligentx 
Can you see the image now? This is the keypad status that the envisalink is reading from the dsc panel, so we know that all those statuses are available to read. This helps to see when the alarm is not arming, that there is an open zone or zone trouble.

![DSC Keypad Status](https://user-images.githubusercontent.com/52211479/92406710-fe971c00-f138-11ea-82f1-19135400621a.png)

**Comment by borjaburgos** [comment]
Hi there! I would also be interested in getting manufacturable formats for the PCB. Something like Gerber would be great. Alternatively, KiCad or EagleCAD would work too. I'm personally interested in the NodeMCU version. 

Thanks!

**Comment by jvitkauskas** [comment]
You can generate Gerber files with Eagle for D1 mini shields https://github.com/taligentx/dscKeybusInterface/tree/master/extras/PCB%20Layouts/WemosD1Mini

I've added my files https://github.com/taligentx/dscKeybusInterface/pull/178

If, for some reason you need lots of boards for cheap you can select panel service and enter 3x4 so you'll get 12 boards on one sheet
![image](https://user-images.githubusercontent.com/9928675/100782945-cf313c00-3415-11eb-95d3-1baf3332967f.png)

**Comment by heytcass** [comment]
> Hi there! I would also be interested in getting manufacturable formats for the PCB. Something like Gerber would be great. Alternatively, KiCad or EagleCAD would work too. I'm personally interested in the NodeMCU version.
> 
> Thanks!

I am also interested in the NodeMCU version. Was anyone able to convert those to a Gerber file? Looking to send to PCBWay.

**Comment by borjaburgos** [comment]
Hey @heytcass I ended up creating my own board, but I went with the D1 mini and an Adafruit MPM360 for power regulation. Let me know if you'd be interested.

<img width="480" alt="Screen Shot 2021-01-15 at 17 31 01" src="https://user-images.githubusercontent.com/3640206/109191827-895de880-7764-11eb-8642-f4e3d4b9d8fe.png">

![A65689E2-BFF4-4A16-A762-226EFB7AF8F2](https://user-images.githubusercontent.com/3640206/109191843-8ebb3300-7764-11eb-982f-399f4ce05d4a.jpg)

**Reply by dreiggy** [reply]
> Hey @heytcass I ended up creating my own board, but I went with the D1 mini and an Adafruit MPM360 for power regulation. Let me know if you'd be interested.
> 
> <img alt="Screen Shot 2021-01-15 at 17 31 01" width="480" src="https://user-images.githubusercontent.com/3640206/109191827-895de880-7764-11eb-8642-f4e3d4b9d8fe.png">
> 
> ![A65689E2-BFF4-4A16-A762-226EFB7AF8F2](https://user-images.githubusercontent.com/3640206/109191843-8ebb3300-7764-11eb-982f-399f4ce05d4a.jpg)

Hey nice work! :) May you share your gerber file?

**Comment by heytcass** [comment]
Hey @borjaburgos , that board looks great! And I also have some D1 minis lying around, so I'd be very interested. Thanks!

**Comment by borjaburgos** [comment]
@heytcass if you live in the US I'm happy to mail you one PCB – I made some extras that I do not need. My email is my GH username at gmail.com.

**Comment by neohusky** [comment]
Hey @borjaburgos 
Any chance you have a spare board to ship down under (AUS).  It would probably be cheaper for me to get it shipped here rather than order a batch.

**Comment by borjaburgos** [comment]
@neohusky shoot me an email
